server/meshmodel/kubernetes/relationships/hierarchical_inventory.json

```diff
@@ -560,6 +560,73 @@
         "from": [],
         "to": []
       }
+    },
+    {
+      "allow": {
+        "from": [
+          {
+            "kind": "Deployment",
+            "model": "kubernetes",
+            "patch": {
+              "patchStrategy": "replace",
+              "mutatorRef": [
+                ["settings", "spec", "selector", "matchLabels"]
+              ], 
+              "description": " Deployments manage ReplicaSets and provide features such as rolling updates and rollback functionality. PDBs can be associated with Deployments to maintain availability during updates and ensure a minimum number of pods are available during the transition."
+            }
+          },
+          {
+            "kind": "ReplicaSet",
+            "model": "kubernetes",
+            "patch": {
+              "patchStrategy": "replace",
+              "mutatorRef": [
+                ["settings", "spec", "selector", "matchLabels"]
+              ],
+              "description": "ReplicaSets ensure a specified number of pod replicas are running at any given time. PDBs can be applied to ReplicaSets to enforce constraints on how many pods can be disrupted at once during scaling operations or node maintenance."
+            }
+          }, 
+          {
+            "kind": "StatefulSet",
+            "model": "kubernetes",
+            "patch": {
+              "patchStrategy": "replace",
+              "mutatorRef": [
+                ["settings", "spec", "selector", "matchLabels"]
+              ],
+              "description": "StatefulSets are used for stateful applications that require stable, unique network identifiers and persistent storage. PDBs can be used with StatefulSets to control pod disruptions and ensure data integrity during maintenance or scaling."
+            }
+          }, 
+          {
+            "kind": "DaemonSet",
+            "model": "kubernetes",
+            "patch": {
+              "patchStrategy": "replace",
+              "mutatorRef": [
+                ["settings", "spec", "selector", "matchLabels"]
```

Comment:
This should be `labels`.
As PDB will acquire/controls pods which have the same labels as specified inside PDB's `selector.matchLabels`

Reply:
Done.